onedal/_config.py

```diff
@@ -21,6 +21,7 @@
 _default_global_config = {
     "target_offload": "auto",
     "allow_fallback_to_host": False,
+    "allow_sklearn_after_onedal": True,
```

Comment:
This will be true by default, but in conformance testing will be switched to False just to check onedal backend full.